closed/custom/Main.gmk

```diff
@@ -119,3 +119,5 @@ ifneq (,$(HEALTHCENTER_JAR))
   # The content must be extracted before module-info can be compiled.
   ibm.healthcenter-java : ibm.healthcenter-copy
 endif # HEALTHCENTER_JAR
+
+openjceplus-copy : openjceplus-libs
```

Comment:
I think more is needed here so that `make clean-openjceplus` will do the right thing.

Reply:
@keithc-ca 

The clean codes, as below, will use the variable, for example "$(JGSKIT_MAKE)",  which defined in Lib.gmk. Do you have any suggestion, for example, how can we add the below clean codes in Lib.gmk, or any better way to get this variable "$(JGSKIT_MAKE)" in the Main.gmk?

```
.PHONY : clean-openjceplus

clean : clean-openjceplus

clean-openjceplus :
	@$(ECHO) Cleaning OpenJCEPlus native
	@$(RM) -f \
		$(addprefix $(JGSKIT_MAKE_PATH)/, \
			com_ibm_crypto_plus_provider_icc_FastJNIBuffer.h \
			com_ibm_crypto_plus_provider_icc_NativeInterface.h \
			libjgskit.*)
	$(MAKE) -C $(JGSKIT_MAKE_PATH) -f $(JGSKIT_MAKE) cleanAll
```